### ISSUE TITLE
Exception when Gradle Sync: Could not find com.karumi:core:2.2.1-SNAPSHOT

### ISSUE THREAD

**Issue by juange87** [Karumi/Shot#62]
### Expected behaviour

Gradle sync works correctly

### Actual behaviour

An exception is thrown when gradle sync is running

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'tipsterchat-android'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find com.karumi:core:2.2.1-SNAPSHOT.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/karumi/core/2.2.1-SNAPSHOT/maven-metadata.xml
       - https://dl.google.com/dl/android/maven2/com/karumi/core/2.2.1-SNAPSHOT/core-2.2.1-SNAPSHOT.pom
       - https://dl.google.com/dl/android/maven2/com/karumi/core/2.2.1-SNAPSHOT/core-2.2.1-SNAPSHOT.jar
       - https://jcenter.bintray.com/com/karumi/core/2.2.1-SNAPSHOT/maven-metadata.xml
       - https://jcenter.bintray.com/com/karumi/core/2.2.1-SNAPSHOT/core-2.2.1-SNAPSHOT.pom
       - https://jcenter.bintray.com/com/karumi/core/2.2.1-SNAPSHOT/core-2.2.1-SNAPSHOT.jar
       - https://repo.maven.apache.org/maven2/com/karumi/core/2.2.1-SNAPSHOT/maven-metadata.xml
       - https://repo.maven.apache.org/maven2/com/karumi/core/2.2.1-SNAPSHOT/core-2.2.1-SNAPSHOT.pom
       - https://repo.maven.apache.org/maven2/com/karumi/core/2.2.1-SNAPSHOT/core-2.2.1-SNAPSHOT.jar
       - http://dl.bintray.com/kotlin/kotlin-eap/com/karumi/core/2.2.1-SNAPSHOT/maven-metadata.xml
       - http://dl.bintray.com/kotlin/kotlin-eap/com/karumi/core/2.2.1-SNAPSHOT/core-2.2.1-SNAPSHOT.pom
       - http://dl.bintray.com/kotlin/kotlin-eap/com/karumi/core/2.2.1-SNAPSHOT/core-2.2.1-SNAPSHOT.jar
       - https://maven.fabric.io/public/com/karumi/core/2.2.1-SNAPSHOT/maven-metadata.xml
       - https://maven.fabric.io/public/com/karumi/core/2.2.1-SNAPSHOT/core-2.2.1-SNAPSHOT.pom
       - https://maven.fabric.io/public/com/karumi/core/2.2.1-SNAPSHOT/core-2.2.1-SNAPSHOT.jar
     Required by:
         project : > com.karumi:shot:3.0.0
```

### Steps to reproduce

Update the version to 3.0.0

### Version of the library

3.0.0

Tested on Android Studio 3.4.2 and 3.5

**Comment by pedrovgs** [comment]
Thanks for reporting @juange87 It has to be related to the latest release. Maybe I forgot to configure the dependencies properly for the release 3.0.0. I won't have time to review this until the middle of September. But we'll take a look.

**Comment by pedrovgs** [comment]
Hi @juange87 I've checked the last release and by mistake part of the artifact was pointing at the wrong version. I've just released Shot 3.0.1 which should fix this issue. Could you please check it out and let me know if it's working as expected now?

**Comment by juange87** [comment]
Everything is working, thank you!
